src/generic/bemparm.c

```diff
@@ -159,7 +159,7 @@ VPUBLIC Vrc_Codes BEMparm_check(BEMparm *thee) {
     }
 
     if (thee->mesh > 2 || thee->mesh < 0) {
-        Vnm_print(2, "BEMparm_check: mesh must be 0 (msms) or 1 and 2 (NanoShaper)");
+        Vnm_print(2, "BEMparm_check: mesh must be 0 or 1 and 2 (NanoShaper)");
```

Comment:
Is a value of `0` still legit since we removed MSMS?  The help message is confusing to me.

Reply:
Good catch, I will update this and the code to be more explicit that only 1 or 2 are valid valued. I updated the documentation to state that 0 (msms) is no longer supported.